### ISSUE TITLE
Show people and number of people removed you since last time you checked

### ISSUE THREAD

**Issue by erickdeoliveiraleal** [tinderjs/tinder-desktop#144]
Show people and number of people removed you since last time you checked, like 6tin 💃

**Comment by mayeaux** [comment]
Good idea for a feature for sure. I'm not actively developing on this project but I'm happy to merge code if someone else is.
